_layouts/projects.html

```diff
@@ -6,68 +6,76 @@
 		<main id="main" role="main">
 			<section class="project id="">
 				<header class="project--header">
+					<p class="project--tags">
+						<span class="project--type">{{ page.type-of-product | default: "N/A"}}</span> ∙ {{ page.tags }}
```

Comment:
Instead of N/A can we make this part not show if type-of-product isn't there? Same with the period after it. Given it doesn't have a label, it's a bit odd to see "N/A" at the top w/o context.